samples/gtk_application.cr

```diff
@@ -1,4 +1,7 @@
-require "gobject/gtk"
+require "../src/gobject"
+require_gobject "Gio"
+require_gobject "Gtk"
+require "../src/gtk"
```

Comment:
I think all of this is because you didn't run the sample from the project root :)